registration/api/v2/serializers.py

```diff
@@ -38,6 +38,46 @@
 logger = logging.getLogger(__name__)
 
 
+class DataSerializer(serializers.Serializer):
+    """
+    Serializer for data that does not need to be persisted directly.
+    Inherit from it to avoid needing to disable abstract-method warnings.
+    """
+
+    def create(self, validated_data):
+        pass
+
+    def update(self, instance, validated_data):
+        pass
+
+
+class GenericObjectSerializer(DataSerializer):
+    """
+    Serializer for any response that returns a JSON dict, without specifying
+    the fields of that dict in detail.
+    """
+    def to_representation(self, instance):
+        return instance
+
+    def to_internal_value(self, data):
+        return data
+
+    class Meta:
+        swagger_schema_fields = {
+            "type": "object",
+            "additionalProperties": True,
+        }
+
+
+class LogoFaviconUploadSerializer(DataSerializer):
+    """
+    Serializer for any response that returns a JSON dict, without specifying
+    the fields of that dict in detail.
+    """
+    logo = serializers.FileField(required=False, use_url=True)
```

Comment:
@giovannicimolin, shouldn't we be using [ImageField](https://www.django-rest-framework.org/api-guide/fields/#imagefield)?